scripts/deploy_assisted_service.sh

```diff
@@ -153,6 +157,10 @@ else
         spawn_port_forwarding_command $SERVICE_NAME $DEBUG_SERVICE_PORT $NAMESPACE $NAMESPACE_INDEX $KUBECONFIG minikube undeclaredip $DEBUG_SERVICE_PORT $DEBUG_SERVICE_NAME
     fi
 
+    add_firewalld_port ${IMAGE_SERVICE_PORT}
+    print_log "Starting port forwarding for deployment/${SERVICE_NAME} on debug port $DEBUG_SERVICE_PORT"
```

Comment:
Should this be `IMAGE_SERVICE_NAME` and `IMAGE_SERVICE_PORT`? Also why `on debug port`?

Reply:
I assume this was just copy-paste from 156